types/phplint/phplint-tests.ts

```diff
@@ -0,0 +1,23 @@
+import { clearCache, cli, gruntPlugin, lint } from 'phplint';
```

Comment:
use `require` instead
```suggestion
import phplint = require("phplint")
```